Sources/CorePayments/Networking/NetworkingClient.swift

```diff
@@ -1,11 +1,10 @@
 import Foundation
 
-// TODO: - Rename to `NetworkingClient`. Now that we have `<PPaaS>API.swift` classes, ths responsibility of this class really is to coordinate networking. It transforms REST & GraphQL into HTTP requests.
 /// This method is exposed for internal PayPal use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
 ///
-/// `APIClient` is the entry point for each payment method feature to perform API requests. It also offers convenience methods for API requests used across multiple payment methods / modules.
+/// `NetworkingClient` transforms REST & GraphQL requests into the proper HTTP format and executes network requests.
 @_documentation(visibility: private)
-public class APIClient {
+public class NetworkingClient {
```

Comment:
🤔 on other name options. Not sure if `NetworkingOperation/Operator`, `NetworkConfiguration`, `NetworkConstructor`, `NetworkBuilder` makes more sense?

Reply:
The methods aren't static, but here's how the definitions would look roughly:
```
Network.fetch(restRequest:_)
NetworkingClient.fetch(restRequest:_)
NetworkBuilder.fetch(restRequest:_)
NetworkOperator.fetch(restRequest:_)
```

Reply:
I think `NetworkOperator ` or `Network/Networking` look nice. But I also don't feel strongly enough about it to not go with `NetworkingClient` if we want. What do you think?

Reply:
Yeh, I agree about not feeling that strongly at the moment. Let's go with `NetworkingClient` and we could always change it again later on if the feeling strikes!